include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -98,6 +98,7 @@ ORT_DEFINE_RELEASE(Value);
 ORT_DEFINE_RELEASE(ModelMetadata);
 ORT_DEFINE_RELEASE(ThreadingOptions);
 ORT_DEFINE_RELEASE(IoBinding);
+ORT_DEFINE_RELEASE(ArenaCfg);
```

Comment:
I think there is a confusion between C++ wrapper objects and C strucurres. This is a C++ header. And C clients can not consume it. Yet, C strcuture ArenaCfg is not defined within c_api.h header.